docs/integrations/wallets.md

```diff
@@ -0,0 +1,108 @@
+</a>
+
+For more examples, click [here](https://github.com/lidofinance/referral-program-integration-examples/tree/main/examples/banners).
+
+### Embedding Staking Widget
+
+If you don't want users to leave your app to stake, you can embed Staking Widget via the `iframe` HTML tag and use your referral link as the source.
+
+Example code,
+```html
+<iframe
+    src="https://stake.lido.fi/?ref=0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B"
+    title="Lido Staking App"
+    height="500"
+    width="100%"
+/>
+```
+
+Preview,
+<iframe
+    src="https://stake.lido.fi/?ref=0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B"
+    title="Lido Staking App"
+    height="500"
+    width="100%"
+/>
+
+For a full example, click [here](https://github.com/lidofinance/referral-program-integration-examples/blob/main/examples/iframe/iframe.html).
+
+### Interacting with Lido contract directly
+
+If none of the options above suits you, you may choose to interact with Lido directly.
+
+[Lido](https://etherscan.io/token/0xae7ab96520de3a18e5e111b5eaab095312d7fe84) provides an easy interface for staking. All you need to do you is call the `submit(_referral)` function, pass your rewards address as the argument and send ether value as the stake.
+
+`web3.js` example,
+
+```javascript
+import Web3 from "web3";
+
+// using partial ABI for simplicity
+const LIDO_ABI =  [
+    {
+        constant: false,
+        inputs: [{ name: "_referral", type: "address" }],
+        name: "submit",
+        outputs: [{ name: "", type: "uint256" }],
+        payable: true,
+        stateMutability: "payable",
+        type: "function"
+    }
+];
+
+const LIDO_ADDRESS = "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84"
+
+const MY_REWARDS_ADDRESS = "0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B"
+
+async function stakeWithLido() {
+    // enable provider
+    const [user] = await window.ethereum.request({
+        method: "eth_requestAccounts"
+    });
+
+    // init web3
+    const web3 = new Web3(window.ethereum);
+    web3.eth.Contract.setProvider(web3);
+
+    // init Lido
+    const lidoContract = new web3.eth.Contract(LIDO_ABI, LIDO_ADDRESS);
+
+    // stake
+    const tx = await lidoContract.methods
+        .submit(MY_REWARDS_ADDRESS)
+        .send({ from: user, value: web3.utils.toWei("0.1") });
+
+    console.log(tx);
+}
+
+```
+
+See [Deployed Contracts](/deployed-contracts) for Lido addresses on Mainnet and testnets.
```

Comment:
Let's add a crosslink to the [stETH/wstETH integration guide](https://docs.lido.fi/guides/steth-integration-guide)